src/popup/components/Modals/AccountCreate.vue

```diff
@@ -7,86 +7,82 @@
     @close="resolve"
   >
     <h2 class="text-heading-2">
-      {{
-        isMultisig
-          ? $t('modals.createAccount.titleMultisig')
-          : $t('pages.accounts.addAccount')
-      }}
+      {{ t('modals.createAccount.title') }}
     </h2>
 
     <p class="message">
-      {{
-        isMultisig
-          ? $t('modals.createAccount.msgMultisig')
-          : $t('modals.createAccount.msg')
-      }}
+      {{ t('modals.createAccount.msg') }}
     </p>
 
     <BtnSubheader
-      v-if="!isMultisig"
-      :header="$t('pages.accounts.addAccount')"
-      :subheader="$t('modals.createAccount.btnSubtitle')"
-      :icon="PlusCircleIcon"
+      :header="$t('common.aeternity')"
+      :subheader="$t('modals.createAccount.btnAESubtitle')"
+      :icon="IconAE"
+      @click="createAEAccount"
+    />
+    <BtnSubheader
+      :header="$t('common.bitcoin')"
+      :subheader="$t('modals.createAccount.btnBTCSubtitle')"
+      :icon="IconBTC"
       :disabled="!isOnline"
-      @click="createPlainAccount()"
+      @click="createBTCAccount"
     />
     <BtnSubheader
-      :header="$t('modals.createMultisigAccount.btnText')"
-      :subheader="$t('modals.createMultisigAccount.btnSubtitle')"
-      :icon="PlusCircleIcon"
+      :header="$t('common.dogecoin')"
+      :subheader="$t('modals.createAccount.btnDOGSubtitle')"
+      :icon="IconDOG"
       :disabled="!isOnline"
-      @click="createMultisigAccount()"
+      @click="createDOGAccount"
     />
-
-    <Loader v-if="loading" />
   </Modal>
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType, ref } from 'vue';
-import { useStore } from 'vuex';
-import { MODAL_MULTISIG_VAULT_CREATE } from '../../utils';
+import { defineComponent, PropType } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { MODAL_AE_ACCOUNT_CREATE } from '../../utils';
 import { useConnection, useModals } from '../../../composables';
 import BtnSubheader from '../buttons/BtnSubheader.vue';
 import Modal from '../Modal.vue';
-import Loader from '../Loader.vue';
-import PlusCircleIcon from '../../../icons/plus-circle-fill.svg?vue-component';
+import IconAE from '../../../icons/coin/aeternity.svg?vue-component';
+import IconBTC from '../../../icons/coin/bitcoin.svg?vue-component';
+import IconDOG from '../../../icons/coin/dogecoin.svg?vue-component';
```

Comment:
I'm not sure about the usage of uppercase letters in file names - couldn't it be standard CamelCase naming convention?